runtime/src/main/java/com/kotcrab/vis/runtime/scene/Scene.java

```diff
@@ -50,36 +46,13 @@
 	private float height;
 
 	/** Used by framework, not indented for external use */
-	public Scene (RuntimeContext context, SceneData data, SceneParameter parameter) {
+	protected Scene(EntityEngineConfiguration engineConfig, SceneData data) {
```

Comment:
I made constructor protected because of your comment "Used by framework, not indented for external use". The best way to prevent external use - encapsulated constructor. More over, I moved some logic that does not concern directly to creation of Scene in SceneLoader. In my opinion, this is more appropriate place for this logic.